assignment3.txt

```diff
@@ -0,0 +1,10 @@
+Ava N.
+June 30, 2016
+
+Assignment 3
+
+1. The code written was invalid because the printf statement should look like this: printf("%d\n",i);
```

Comment:
Correct! You'd also need to initialize i to a number greater than 0 beforehand for the program to properly run.